runtime/manual_tests/loader-tests.js

```diff
@@ -18,19 +18,19 @@ describe('loader', function() {
   it('correctly loads Thing as a dependency', async () => {
     let schemaString = await loader.loadResource('http://schema.org/Product');
     let manifest = await Manifest.parse(schemaString, {loader, fileName: 'http://schema.org/Product'});
-    assert(manifest.schemas.Product.fields.description == 'Text');
+    assert.equal(manifest.schemas.Product.fields.description, 'Text');
   }).timeout(10000);
 
   it('can read a schema.org schema that aliases another type', async () => {
     let schemaString = await loader.loadResource('http://schema.org/Restaurant');
     let manifest = await Manifest.parse(schemaString, {loader, fileName: 'http://schema.org/Restaurant'});
-    assert(manifest.schemas.Restaurant.fields.servesCuisine == 'Text');
+    assert.equal(manifest.schemas.Restaurant.fields.servesCuisine, 'Text');
   }).timeout(10000);
 
   it('can read a schema.org schema with multiple inheritance', async () => {
     let schemaString = await loader.loadResource('http://schema.org/LocalBusiness');
     let manifest = await Manifest.parse(schemaString, {loader, fileName: 'http://schema.org/LocalBusiness'});
-    assert(manifest.schemas.LocalBusiness.fields.duns == 'Text');
-    assert(manifest.schemas.LocalBusiness.fields.branchCode == 'Text');
+    assert.equal(manifest.schemas.LocalBusiness.fields.duns, 'Text');
+    assert(manifest.schemas.LocalBusiness.fields.branchCode, 'Text');
```

Comment:
`.equal` is missing here

Reply:
Thanks for spotting that!

Pushed a new change and updated the PR.